test/files/rules/indentwith_tabs.test.ts

```diff
@@ -1,5 +1,8 @@
 // valid code
 module TestModule {
+	/**
+	 * Some documentation
+	 */
```

Comment:
This seems to overlap somewhat with the jsdoc rule -- I'm not sure that it makes sense for the indent rule to be aware of jsdoc comments vs other kinds -- and jsdocs aren't the only kind of multiline comments. I think it's likely that simply ignoring indentation inside multiline comments makes the most sense, and is most correct.